### ISSUE TITLE
regression on k8s v1.9.4, configMap volume can't be used until fix in v1.9.5

### ISSUE THREAD

**Issue by lakano** [kubernetes/minikube#2642]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG

The k8s v1.9.4 contain a regression that doesn't works with configMap volume ( https://github.com/kubernetes/kubernetes/issues/61076 ). It's fixed in v1.9.5, is it possible to add support for v1.9.5 or v1.9.6 please?

Please provide the following details:

**Environment**:

**Minikube version** (use `minikube version`):
- **OS** (e.g. from /etc/os-release):
- **VM Driver** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep DriverName`):
- **ISO version** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep -i ISO` or `minikube ssh cat /etc/VERSION`):
- **Install tools**:
- **Others**:
The above can be generated in one go with the following commands (can be copied and pasted directly into your terminal):
```
minikube version v0.25.2
echo "";
echo "OS:";
cat /etc/os-release
echo "";
echo "VM driver": 
grep DriverName ~/.minikube/machines/minikube/config.json
echo "";
echo "ISO version";
grep -i ISO ~/.minikube/machines/minikube/config.json
```

**What happened**:
The default k8s version (v1.9.4) installed include a regression on configMap volume usage.

**What you expected to happen**:
Use the last stable version of k8s please.

**How to reproduce it** (as minimally and precisely as possible):
example on the 1st post: https://github.com/kubernetes/kubernetes/issues/61076

**Output of `minikube logs` (if applicable)**:

**Anything else do we need to know**:
BTW, setting the k8s version to "v1.9.0" on minikube start doesn't seems to work ( I'll dig again and report a new issue for this ) EDIT: It's works but it's automatically upgrade to v1.9.4 on next start if not added in flag, and you can't downgrade so, the best option is to keep it in the config: minikube config set kubernetes-version v1.9.0

**Comment by kindermoumoute** [comment]
I keep using `minikube version v0.25.0` for this reason.

**Comment by attila123** [comment]
@dlorenc Could you please help with this? I saw you updated to 1.9.4, and it does not seem a trivial commit, but you have the needed expertise. :)

**Comment by MichaelScript** [comment]
This is fixed in minikube v.0.26.0

```
curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.26.0/minikube-darwin-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
```

**Comment by lakano** [comment]
Thanks

**Comment by podviaznikov** [comment]
<img width="729" alt="1__anton_podviaznikov_____zsh_" src="https://user-images.githubusercontent.com/429706/43739267-4fe7f6ea-997c-11e8-8e48-662199ca494e.png">
For some reason I don't see it's fixed.
k8s 1.9.6 is not available as a version, while 1.9.4 with `subpath` problem is there
